compiler/rustc_mir/src/transform/check_unsafety.rs

```diff
@@ -564,17 +563,10 @@ fn is_enclosed(
     if parent_id != id {
         if used_unsafe.contains(&parent_id) {
             Some(("block".to_string(), parent_id))
-        } else if let Some(Node::Item(&hir::Item {
-            kind: hir::ItemKind::Fn(ref sig, _, _), ..
-        })) = tcx.hir().find(parent_id)
+        } else if let Some(Node::Item(&hir::Item { kind: hir::ItemKind::Fn(..), .. })) =
+            tcx.hir().find(parent_id)
         {
-            if sig.header.unsafety == hir::Unsafety::Unsafe
-                && !tcx.features().unsafe_block_in_unsafe_fn
-            {
-                Some(("fn".to_string(), parent_id))
-            } else {
-                None
-            }
+            None
```

Comment:
I think this is incorrect and leading to the current CI failure

Reply:
@LeSeulArtichaut did you plan to do further work on this PR?

Reply:
Yes, sorry, I need to get back to it...

Reply:
No worries, just wanted to know the status. :)